package/MDAnalysis/converters/RDKitParser.py

```diff
@@ -303,8 +299,8 @@ def parse(self, **kwargs):
         if atomtypes:
             attrs.append(Atomtypes(np.array(atomtypes, dtype=object)))
         else:
-            atomtypes = guessers.guess_types(names)
```

Comment:
I removed all type guessing from parsers and transferred it to be guessed at the universe creation. For PDB, FIAMS, XYZP and RDKit parsers, I assigned the `atomtypes` to be either read from the file if not be assigned to elements,  if not be assigned to names (as per issue #[2931](https://github.com/MDAnalysis/mdanalysis/pull/2931)). but if we are not ready for this move I'll remove it no worries

Reply:
It is OK. I just wanted to avoid missing something. We should keep note of that and maybe have it a separate entry in the changelog.

We will also need to be careful with the tests.